include/mbedtls/aes.h

```diff
@@ -139,23 +139,24 @@ void mbedtls_aes_free( mbedtls_aes_context *ctx );
  *                 It must be the first API called before using
  *                 the context.
  *
- * \param ctx      The AES XTS context to initialize.
+ * \param ctx      The AES XTS context to initialize. Must not be NULL.
  */
 void mbedtls_aes_xts_init( mbedtls_aes_xts_context *ctx );
 
 /**
  * \brief          This function releases and clears the specified AES XTS context.
  *
- * \param ctx      The AES XTS context to clear.
+ * \param ctx      The AES XTS context to clear. If NULL, no action is taken.
  */
 void mbedtls_aes_xts_free( mbedtls_aes_xts_context *ctx );
 #endif /* MBEDTLS_CIPHER_MODE_XTS */
 
 /**
  * \brief          This function sets the encryption key.
  *
- * \param ctx      The AES context to which the key should be bound.
- * \param key      The encryption key.
+ * \param ctx      The AES context to which the key should be bound. Must not
+ *                 be NULL.
+ * \param key      The encryption key. Must not be NULL.
```

Comment:
Minor: Use `\c NULL`.

Reply:
I think we should state a more precise precondition here, even if what we can effectively check for is much less. Specifically, I'd suggest to say something along the lines of `Must be a readable buffer of length \p keybits bits.`.

Reply:
In general, the documentation should state function preconditions are precise as possible, even if the parameter validation implementation can only check rough approximations.

Reply:
In a similar vain, I'd prefer to say "The AES context to which the key should be bound. Must be initialized.` about the precondition on the context.